### ISSUE TITLE
[FE] My VA Forms Status: Create accordion component for "what if I don't see my application"

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#89642]
## Background
In the new "Benefit applications and forms" section on My VA we'll be including explainer text to tell the user why they may not see certain forms there as not all forms are included in the forms status mvp. We need to create an accordion utilizing the accordion component in the design system that will appear below all cards in the Benefit applications and forms section on My VA.

## Tasks
- [x] Review design goals/intent with Matt as needed
- [x] Pull in accordion web component from design system
- [x] Place behind existing feature flag for forms status work (`my_va_form_submission_statuses`)
- [x] Copy/paste content from FE documentation file or figma file
- [x] Create unit tests 

## Acceptance Criteria
- [x] Accordion component exists in Benefit applications and forms section on My VA that expands when the user clicks it. The accordion is always present even when the user has no forms.
- [x] Accordion component exists in Benefit applications and forms section on My VA regardless of whether they have drafts or "in progress" forms
- [x] The component is behind the same feature flag as the rest of the forms status work (`my_va_form_submission_statuses`)

## Validation
- _Who can validate this ticket? (FE, BE, Design, PM)?_ Any team member
- _How can this work be validated?_ Navigate to staging and confirm accordion exists and functions as intended

## Resources
- [Benefit intake API documentation](https://developer.va.gov/explore/api/benefits-intake/docs?version=current)
- [FE technical discovery](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/forms-status-on-My-VA/technical-discovery.md#frontend)
- [Figma file for accordion](https://www.figma.com/design/15yOY4VEzitxm5tRMDiAzz/My-VA?node-id=1237-25463&t=1lxbzcjRIZi8FDKs-4)

**Comment by ACParker89** [comment]
Discussing post MVP potentials:  @DanielMiller-agile6 @mattmarino-adhoc will add tickets for backlog for design and engineering needs for [error states ](https://dsva.slack.com/archives/C909ZG2BB/p1724174029103339)

**Comment by ACParker89** [comment]
@mattmarino-adhoc @DanielMiller-agile6 - do we need follow on tickets for this still or are we all set?

**Comment by DanielMiller-agile6** [comment]
We should sync up today on any additional error states for UI we might need.  I'll need to add an additional logging call so we can track when submitted forms are loaded successfully, but the form statuses from LH fail... I'm not sure what we'd like to do in this case yet, but at the very least we'll need to log this when it happens.  Currently, what will happen is that "Draft" forms may show, but no "Submitted" forms will, and no indication will be available to the user aside from the usual help text in accordion.

**Comment by ajakabcin** [comment]
@DanielMiller-agile6 and @mattmarino-adhoc I'm going to set up a call with you me, Allison and Tom for later this week to talk about the error states.

**Comment by ajakabcin** [comment]
hey @DanielMiller-agile6 reopening this - I noticed two super small typos in the copy within the accordion 

1. there should be a space between "at" and the phone number
2. I think there should be a closed parenthesis after 711

I found this using firefox and user 100.

![Screenshot 2024-08-27 at 3 02 37 PM](https://github.com/user-attachments/assets/b6a9971f-60da-493e-a728-932a62d7dc03)
 
cc @mattmarino-adhoc not sure if this is also in our copy elsewhere just fyi!
